target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -29,7 +31,9 @@ TARGET_DEVICES += sun7i-a20-olinuxino-lime2
 
 
 define Device/sun7i-a20-olinuxino-lime2-emmc
-  DEVICE_TITLE:=Olimex A20-OLinuXino-LIME2-eMMC
+  DEVICE_VENDOR := Olimex
+  DEVICE_MODEL := A20-OLinuXino-LIME2
+  DEVICE_VARIANT := eMMC
```

Comment:
There is a eMMC and a SDCard variant, but the SDCard variant does not have a DEVICE_VARIANT is that ok?

Reply:
I guess it'd be better to add SDCard as variant...

Reply:
Official product page (for reference): https://www.olimex.com/Products/OLinuXino/A20/A20-OLinuXino-LIME2/open-source-hardware

From what I understood, the new variables should be used to differentiate between hardware variants, not firmware.

The eMMC variant for this board also has an SDCard slot, so in theory it should be able to boot the other image too. In fact, looking at its device tree, it only adds and additional mmc node.

This issue is applicable to all devices that have two or more storage location where OpenWRT could be installed

I think @aparcar and @adrianschmutzler were discussing how to handle this. Have you reached a consensus?

Reply:
Having said all that. I have no problem on adding SDCard to the other variant. I just want consistency on the names

Reply:
Just to answer the question: From my point of view, we have not reached a clear consensus here, so it is somehow left to the gut feeling of author and reviewer. However, I can live with this being not regulated for now.